src/database.test.cpp

```diff
@@ -0,0 +1,15 @@
+#include "gtest/gtest.h"
+#include <iostream>
+#include <string>
+#include <silo/database.h>
+#include <silo/preprocessing_config.h>
+
+TEST(database_test, should_build_database_without_errors) {
+
+   const std::string input_directory("./testBaseData/");
+   const std::string output_directory("./build/");
+   auto config = silo::PreprocessingConfig(input_directory, output_directory, "minimal_set.tsv", "minimal_set.fasta");
+   auto database = std::make_shared<silo::Database>(input_directory);
+
+   EXPECT_NO_THROW(database->preprocessing(config));
```

Comment:
We should instead check that the database is not empty. Spontaneous idea: `silo::Database::get_db_info()` should do the job.